netlogo-gui/src/main/prim/etc/_extractrgb.java

```diff
@@ -3,19 +3,38 @@
 package org.nlogo.prim.etc;
 
 import org.nlogo.api.LogoException;
+import org.nlogo.api.AgentException;
+import org.nlogo.api.LogoListBuilder;
 import org.nlogo.core.LogoList;
 import org.nlogo.core.Syntax;
+import org.nlogo.core.SyntaxJ;
 import org.nlogo.nvm.Context;
 import org.nlogo.nvm.Reporter;
 
 public final strictfp class _extractrgb
     extends Reporter {
 
-
   @Override
   public Object report(final org.nlogo.nvm.Context context) throws LogoException {
-
-    return report_1(context, argEvalDoubleValue(context, 0));
+    try{
```

Comment:
Prefer space after keyword and before opening curly bracket.  There are other occurrences of this, but I'll just leave this comment.